test/bdd/go.mod

```diff
@@ -20,10 +20,10 @@ require (
 	github.com/golang/snappy v0.0.3 // indirect
 	github.com/google/uuid v1.1.2
 	github.com/gorilla/mux v1.8.0
-	github.com/hyperledger/aries-framework-go v0.1.6-0.20210409151411-eeeb8508bd87
-	github.com/hyperledger/aries-framework-go/component/storage/leveldb v0.0.0-20210421205521-3974f6708723
-	github.com/hyperledger/aries-framework-go/component/storageutil v0.0.0-20210409151411-eeeb8508bd87
-	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210421205521-3974f6708723
+	github.com/hyperledger/aries-framework-go v0.1.7-0.20210422133815-2ef2d99cb692
+	github.com/hyperledger/aries-framework-go/component/storage/leveldb v0.0.0-20210422133815-2ef2d99cb692
+	github.com/hyperledger/aries-framework-go/component/storageutil v0.0.0-20210422133815-2ef2d99cb692
+	github.com/hyperledger/aries-framework-go/spi v0.0.0-20210422133815-2ef2d99cb692
```

Comment:
also here

Reply:
Actually, these imports aren't in go.sum to begin with. I assume this is due to the replace.